src/components/Breadcrumb/Breadcrumb.vue

```diff
@@ -100,6 +103,13 @@ export default {
 			return this.to ? 'router-link' : 'a'
 		},
 	},
+	beforeMount() {
+		// filter invalid items
+		ValidateSlot(this.$slots.default, ['Actions'], this)
```

Comment:
Ah, you should not bother here, just already put the Actions inside, wrapping the slot.
That way the Actions component will take care of it.
```vue
<BreadCrumb>
	<ActionButton />
</BreadCrumb>
```

Like we do in the Sidebar for example :)
https://github.com/nextcloud/nextcloud-vue/blob/ed35e7754341b1e0497141fa66743994b3b8b735/src/components/AppSidebar/AppSidebar.vue#L150

Reply:
This is still not done :)

Reply:
> 
> 
> This is still not done :)

What do you mean? I removed the `ValidateSlot` function from the component.

Reply:
Ah, do you mean, I should wrap the slot within an `Actions`, so the user only has to provide the `ActionButton` etc.?

Reply:
I fixed that and forwarded the force-menu prop.